plugins/rspamd.js

```diff
@@ -159,6 +159,13 @@ exports.parse_response = function (rawData, connection) {
         return;
     }
 
+    if (Object.keys(data).length === 1 && data.error) {
+        connection.transaction.results.add(plugin, {
```

Comment:
Errors stored to results are automatically emitted as log error, so that is preferable.